### PR TITLE
Backend Setup #6

### LINKED ISSUE #6: Setup backend "Hoggar"
# Description
Setup an express application for the backend 

## acceptance criteria
- code name : `hoggar`
- express with type script
- commands  dev:hoggar, build:hoggar, run:hoggar on package.json 
- simple GET / saying hello world

### PR DESCRIPTION
**Pull request by badido39**
server created with express based Type -Script .add dev dependeny in Main package.json 
closes #6

### CODE REVIEW

File: hoggar/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+PORT=8000
```
Comment (nazimboudeffa) [inline hoggar/.env]: Can you add .env.sample instead of directly .env ?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+.env
```
Comment (osminosm) [inline .gitignore]: this ignores `.env` but not `hoggar/.env`
maybe use a wildcard like **/.env

File: hoggar/.env (hunk 1 above)
Comment (osminosm) [inline hoggar/.env]: this file is not ignored,

but there should an exmaple

File: hoggar/dist/index.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+"use strict";
```
Comment (osminosm) [inline hoggar/dist/index.js]: ./dist should be ignored

**Review by raxrs — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (badido39) [reply .gitignore]: done

File: hoggar/.env [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+##GITHUB_
+CLIENT_ID=59bacf304331ecf0c890
+
+CLIENT_SECRET=2bff573643586d42ac418b3444d3efd3c42f26a6
```
Comment (badido39) [inline hoggar/.env]: even it ignored ....

**Review by badido39 — COMMENTED** [review]
.

**Review by nazimboudeffa — APPROVED** [review]

File: hoggar/.env (hunk 1 above)
Comment (badido39) [reply hoggar/.env]: yes , i am working on it

File: hoggar/dist/index.js [hunk 5]
```diff
@@ -0,0 +1,110 @@
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```
Comment (osminosm) [inline hoggar/dist/index.js]: should not be committed

**Review by osminosm — APPROVED** [review]

**Pull request merged**